src/components/PostPageDetail/elements/Content.js

```diff
@@ -227,7 +229,7 @@ const Content = ({
             </View>
           </View>
         ) : null}
-        {item && item.post_type === POST_TYPE_LINK && (
+        {item && item.post_type === POST_TYPE_LINK && images_url?.length === 0 ? (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The added condition to check if `images_url` array exists and is not empty before rendering the link post is a good practice. This prevents unnecessary rendering when there are no images to display.
```diff
-        {item && item.post_type === POST_TYPE_LINK && (
+        {item && item.post_type === POST_TYPE_LINK && images_url?.length === 0 ? (
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->